PyPDF2/generic.py

```diff
@@ -200,7 +200,7 @@ def writeToStream(
         self.write_to_stream(stream, encryption_key)
 
     @staticmethod
-    def read_from_stream(stream: StreamType, pdf: Any) -> "ArrayObject":  # PdfReader
+    def read_from_stream(stream: StreamType, pdf: Any, forcedEncoding: Union[None,str,List[str],Dict[int,str]] = None) -> "ArrayObject":  # PdfReader
```

Comment:
forcedEncoding seems never to be assigned. I guess that is one part that is still a TODO? What does this do?